Loungeware/scripts/___LW_FGameLoader/___LW_FGameLoader.gml

```diff
@@ -346,36 +346,20 @@ function LW_FGameLoaderColourTransformer(field_name, default_value) : LW_FGameLo
 	
 	_get_value_internal  = function(colour){ 
 		if(is_string(colour)){
+			// Remove prefix
 			var first_char = string_char_at(colour, 1);
 			if (first_char == "#" || first_char == "$")
-				colour = string_delete(colour, 0, 1);
+				colour = string_delete(colour, 1, 1);
 			if (string_copy(colour, 1, 2) == "0x")
-				colour = string_delete(colour, 0, 2);
-			
-			var result=0;
-
-			// special unicode values
-			var ZERO=ord("0");
-			var NINE=ord("9");
-			var A=ord("A");
-			var F=ord("F");
-
-			for (var i=1; i<=string_length(colour); i++){
-			    var c=ord(string_char_at(string_upper(colour), i));
-			    // you could also multiply by 16 but you get more nerd points for bitshifts
-			    result=result<<4;
-			    // if the character is a number or letter, add the value
-			    // it represents to the total
-			    if (c>=ZERO&&c<=NINE){
-			        result=result+(c-ZERO);
-			    } else if (c>=A&&c<=F){
-			        result=result+(c-A+10);
-			    // otherwise complain
-			    }
-			}
-			
-			return result;
-			
+				colour = string_delete(colour, 1, 2);
+				
+			// Make the colour
+			colour = string_upper(colour);
+			var dec = 0, h = "0123456789ABCDEF", p;
+			for (p = 1; p <= string_length(colour); p += 1) {
+		        dec = dec << 4 | (string_pos(string_char_at(colour, p), h) - 1);
+		    }
+		    return (dec & 16711680) >> 16 | (dec & 65280) | (dec & 255) << 16;
```

Comment:
probably would have been better to use `0xFF0000` instead of `16711680`, same for the others:
 - `16711680` -> `0xFF0000`
 - `65280` -> `0x00FF00`
 - `255` -> `0x0000FF`